### PR TITLE
Update print instructions in README.md

### PR DESCRIPTION
**Pull request by RocketUnicorn**
Added printing instructions paragraph related to the M2 machine.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
 - Material Safety Data Sheets ([English](docs/msds-en.md) and [French](docs/msds-fr.md)) are available
 - [Assembly instructions](docs/assembly.md)
 
+## Printing Instructions
+Theses printing instruction are meant for printing on a AON-M2 or an AON-M2-2020.
```
Comment (mdtusz) [inline README.md]:
```suggestion

The following instructions are intended for printing using the AON-M2 or an AON-M2-2020.
```

File: README.md [hunk 2]
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+
+### Printer Preparation
+1. Install a 0.6 mm nozzle heater block assembly. Instructions [here](https://docs.aon3d.com/maintenance/replace_hba/).
```
Comment (mdtusz) [inline README.md:160]:
```suggestion

1. Install a 0.6 mm nozzle heater block assembly. Instructions [here](https://docs.aon3d.com/maintenance/replace_hba/). 
```

File: README.md [hunk 3]
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
 - Material Safety Data Sheets ([English](docs/msds-en.md) and [French](docs/msds-fr.md)) are available
 - [Assembly instructions](docs/assembly.md)
 
+## Printing Instructions
+Theses printing instruction are meant for printing on a AON-M2 or an AON-M2-2020.
+
+### Printer Preparation
+1. Install a 0.6 mm nozzle heater block assembly. Instructions [here](https://docs.aon3d.com/maintenance/replace_hba/). 
+2. Install the aluminum build plate. Ensure that the Kapton tape is in good condition on all the surfaces. (If not, follow [this procedure](https://docs.aon3d.com/maintenance/kapton_application/)to replace it.
```
Comment (mdtusz) [inline README.md]:
```suggestion
2. Install the aluminum build plate. Ensure that the Kapton tape is in good condition on all the surfaces. (If not, follow [this procedure](https://docs.aon3d.com/maintenance/kapton_application/) to replace it. 
```

File: README.md [hunk 4]
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+3. Clean the surface with IPA (isopropyl alcohol) 70% concentration minimum.
+4. Load ABS in the 0.6 mm toolhead you want to use. Instructions [here](https://docs.aon3d.com/operation/material_switch/).
+5. Preheat the machine to the following temperatures:
+
+| Component  | Temperature |  
+|:----------:|:-----------:|
+| Extruder   | 250°C       |  
+| Build platform  | 105°C  | 
+| Chamber    | 60°C        | 
+6. Once the chamber has reached thermal equilibrium, initiate the probing sequence on the entire bed  and set the Z offset. Instructions [here](https://docs.aon3d.com/calibration/z_calibration/).
```
Comment (mdtusz) [inline README.md:173]:
```suggestion

6. Once the chamber has reached thermal equilibrium, initiate the probing sequence on the entire bed  and set the Z offset. Instructions [here](https://docs.aon3d.com/calibration/z_calibration/).
```

File: README.md [hunk 5]
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+
+
+### Frame Printing
+
+We have included two generated G-codes that are ready to upload in your machine for you to start to print as soon as possible. They are intended to be used with ABS on a 0.6 mm nozzle. One of the G-code is for T0 (left toolhead), the other one is for T1 (right toolhead).
```
Comment (mdtusz) [inline README.md]:
```suggestion
We have included two generated G-code files that are ready for use on the AON-M2 and AON-M2-2020.
They are intended for use with ABS and a 0.6 mm nozzle.
One G-code file is specific for printing with T0 (left toolhead) and the other is for use with T1 (right toolhead).
```

File: README.md [hunk 6]
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+
+If you wish to modify the files, we also included the factory files to use with Simplify 3D. However, our settings have been tested and any modification can cause print defects.
```
Comment (mdtusz) [inline README.md]:
```suggestion
If you wish to modify the files, the original factory files for use with Simplify 3D have been included.
However, the specified settings have been tested and any modification may cause print defects.
```

File: README.md [hunk 7]
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+
+Each print will get you 5 face shield frames.
```
Comment (mdtusz) [inline README.md]:
```suggestion
Each print will yield 5 face shield frames. 
```

File: README.md [hunk 8]
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+How to remove the part from the build platform. It is suggested to proceed as if you were diagnosed with the virus and were very cautious about not transmitting it.
+
+- Wear a mask.
+- Wear latex gloves, sanitize your hands with IPA.
+- Sanitize the printer’s door and screen with IPA.
+- Sanitize a spatula scraper with IPA.
+- Lower the bed and move the toolhead so you can access the parts easily. **The parts and machines might be hot!**
+- Remove the skirt (single outline of material).
+- With the scraper, carefully detach the extremity of the face shield frame. This will initiate the rest of the removal. Gently detach the part by placing the scraper under the part.
+- Place the parts in a sanitized bag for storage.
```
Comment (mdtusz) [inline README.md:210]: This section is a bit of a duplicate of what is written [here](https://github.com/aon3d/covid-face-shield/pull/12/files#diff-04c6e90faac2675aa89e2176d2eec7d8R73). Perhaps an edit or two to make it more specific, or migrate the handling procedures down here.

**Review by mdtusz — COMMENTED** [review]
Good additions - just some minor wording and voice changes. I'll let @nugvivi give the final review before full approval. 

:+1:

File: README.md [hunk 9]
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
 - Material Safety Data Sheets ([English](docs/msds-en.md) and [French](docs/msds-fr.md)) are available
 - [Assembly instructions](docs/assembly.md)
 
+## Printing Instructions
```
Comment (engnadeau) [inline README.md:152]:
```suggestion
## Printing Instructions

```

linebreak between heading and paragraph for markdown

File: README.md [hunk 10]
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+Theses printing instruction are meant for printing on a AON-M2 or an AON-M2-2020.
+
+### Printer Preparation
```
Comment (engnadeau) [inline README.md:157]:
```suggestion
### Printer Preparation

```

linebreak between heading and paragraph for markdown

File: README.md [hunk 11]
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+1. Install a 0.6 mm nozzle heater block assembly. Instructions [here](https://docs.aon3d.com/maintenance/replace_hba/). 
+2. Install the aluminum build plate. Ensure that the Kapton tape is in good condition on all the surfaces. (If not, follow [this procedure](https://docs.aon3d.com/maintenance/kapton_application/)to replace it. 
+3. Clean the surface with IPA (isopropyl alcohol) 70% concentration minimum.
```
Comment (engnadeau) [inline README.md]:
```suggestion
3. Clean the surface with IPA (isopropyl alcohol), minimum concentration of 70% for disinfection purposes.
```

File: README.md [hunk 12]
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+4. Load ABS in the 0.6 mm toolhead you want to use. Instructions [here](https://docs.aon3d.com/operation/material_switch/).
+5. Preheat the machine to the following temperatures:
+
+| Component  | Temperature |  
+|:----------:|:-----------:|
+| Extruder   | 250°C       |  
+| Build platform  | 105°C  | 
+| Chamber    | 60°C        |
```
Comment (engnadeau) [inline README.md:170]:
```suggestion
| Chamber    | 60°C        | 

```

File: README.md [hunk 13]
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+We have included two generated G-codes that are ready to upload in your machine for you to start to print as soon as possible. They are intended to be used with ABS on a 0.6 mm nozzle. One of the G-code is for T0 (left toolhead), the other one is for T1 (right toolhead).
+
+If you wish to modify the files, we also included the factory files to use with Simplify 3D. However, our settings have been tested and any modification can cause print defects.
+
+Each print will get you 5 face shield frames. 
+
+Plan 112g of ABS filament and 2 hours of print time per print.
+
+
+### Frame Removal
```
Comment (engnadeau) [inline README.md:201]:
```suggestion
### Frame Removal

```

**Review by engnadeau — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (RocketUnicorn) [reply README.md:210]: done in : 0d407b5
https://github.com/aon3d/covid-face-shield/pull/12/commits/0d407b5170eecbc5f6190cfadee4c6f8afad5ad7

**Review by RocketUnicorn — COMMENTED** [review]
Thanks for your feedback! 
I integrated all your suggestions :)

File: README.md [hunk 14]
```diff
@@ -124,6 +125,53 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
 - Material Safety Data Sheets ([English](docs/msds-en.md) and [French](docs/msds-fr.md)) are available
 - [Assembly instructions](docs/assembly.md)
 
+## Printing Instructions
+
+
+The following instructions are intended for printing using the AON-M2 or an AON-M2-2020.
```
Comment (nugvivi) [inline README.md]:
```suggestion
The following instructions are intended for printing using the AON-M2 or AON-M2 2020.
```

File: README.md [hunk 15]
```diff
@@ -124,6 +125,53 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+
+### Printer Preparation
+
+
+1. Install a 0.6 mm nozzle heater block assembly. Instructions [here](https://docs.aon3d.com/maintenance/replace_hba/). 
+2. Install the aluminum build plate. Ensure that the Kapton tape is in good condition on all the surfaces. (If not, follow [this procedure](https://docs.aon3d.com/maintenance/kapton_application/) to replace it. 
+3. Clean the surface with IPA (isopropyl alcohol), minimum concentration of 70% for disinfection purposes.
```
Comment (nugvivi) [inline README.md:162]: Apply Dimafix or hairspray as well - ABS doesn't stick well enough to clean Kapton without brims. We don't want to use brims so we don't have to post-process

File: README.md [hunk 16]
```diff
@@ -124,6 +125,53 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+4. Load ABS in the 0.6 mm toolhead you want to use. Instructions [here](https://docs.aon3d.com/operation/material_switch/).
+5. Preheat the machine to the following temperatures:
+
+| Component  | Temperature |  
+|:----------:|:-----------:|
+| Extruder   | 250°C       |
```
Comment (nugvivi) [inline README.md]: Don't preheat the hot end yet

File: README.md [hunk 17]
```diff
@@ -124,6 +125,53 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+5. Preheat the machine to the following temperatures:
+
+| Component  | Temperature |  
+|:----------:|:-----------:|
+| Extruder   | 250°C       |  
+| Build platform  | 105°C  | 
+| Chamber    | 60°C        | 
+
+
+6. Once the chamber has reached thermal equilibrium, initiate the probing sequence on the entire bed  and set the Z offset. Instructions [here](https://docs.aon3d.com/calibration/z_calibration/).
```
Comment (nugvivi) [inline README.md:173]: Use these probe bounds:
Left: 15
Right: 435
X Count: 10
Front: 15
Back/Rear: 435
Y Count: 10

File: README.md [hunk 18]
```diff
@@ -124,6 +125,53 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+We have included two generated G-code files that are ready for use on the AON-M2 and AON-M2-2020.
+They are intended for use with ABS and a 0.6 mm nozzle.
+One G-code file is specific for printing with T0 (left toolhead) and the other is for use with T1 (right toolhead).
+
+If you wish to modify the files, the original factory files for use with Simplify 3D have been included.
+However, the specified settings have been tested and any modification may cause print defects.
+
+Each print will yield 5 face shield frames. 
+
+Plan 112g of ABS filament and 2 hours of print time per print.
```
Comment (nugvivi) [inline README.md]:
```suggestion
Plan 112g of ABS filament and just over 2 hours of print time per print.
```

File: README.md [hunk 19]
```diff
@@ -124,6 +125,53 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+
+
+### Frame Removal
+
+How to remove the part from the build platform. Proceed following the aformentioned [handling(#handling) guidelines.
```
Comment (nugvivi) [inline README.md]:
```suggestion
How to remove the part from the build platform. Proceed following the aformentioned [handling](#handling) guidelines. 
```
i think this needs the second square bracket?

File: README.md [hunk 20]
```diff
@@ -124,6 +125,53 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+Each print will yield 5 face shield frames. 
+
+Plan 112g of ABS filament and 2 hours of print time per print.
+
+
+### Frame Removal
+
+How to remove the part from the build platform. Proceed following the aformentioned [handling(#handling) guidelines. 
+
+- Lower the bed and move the toolhead so you can access the parts easily. **The parts and machines might be hot!**
```
Comment (nugvivi) [inline README.md]: The gcode files and factory file should have a command in the ending script to move the tools to the back of the machine

Comment (nugvivi) [reply README.md]: the bed and chamber temps are also left on at the end for quick restart

File: README.md (hunk 16 above)
Comment (RocketUnicorn) [reply README.md]: Removed the extruder line in the table

File: README.md (hunk 17 above)
Comment (RocketUnicorn) [reply README.md:173]: Added a table with these values

File: README.md (hunk 20 above)
Comment (RocketUnicorn) [reply README.md]: Don't you think leave the chamber on, opening the door and removing the part will cause thermal runaway issues?

Comment (RocketUnicorn) [reply README.md]: Anyway, I will raise an issue about this but for now I will include your instructions.

File: README.md (hunk 15 above)
Comment (RocketUnicorn) [reply README.md:162]: Noted! It will be in the next version.

**Review by engnadeau — DISMISSED** [review]

File: README.md [hunk 21]
```diff
@@ -101,7 +104,6 @@ As it seems a person can carry the virus and be asymptomatic, it is best to act
 - Wash your hands before removing prints
 - Wipe down tools with IPA before using them to remove prints
 - Wear a mask while handling prints
-- Touch points on printers (anywhere you might touch the machine) are wiped down regularly with IPA every 4h (e.g., twice a shift)
```
Comment (nugvivi) [inline README.md:104]: Why remove this?

Comment (nugvivi) [inline README.md:104]: Why remove this?

File: README.md [hunk 22]
```diff
@@ -147,6 +149,66 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
...
+Plan 112g of ABS filament and just over 2 hours of print time per print.
+
+
+### Frame Removal
+
+How to remove the part from the build platform. Proceed following the aformentioned [handling](#handling) guidelines. 
+
+**The parts and machines are hot!**
+- At the end of the print, the toolheads will move to the back of the machine for easy access to the printed frames.
+- The toolheads, the build platform and the chamber will remain heated to allow for a quick restart.
```
Comment (nugvivi) [inline README.md]:
```suggestion
- The build platform and the chamber will remain heated to allow for a quick restart.
```

**Review by nugvivi — CHANGES_REQUESTED** [review]
minor correction

File: README.md (hunk 21 above)
Comment (RocketUnicorn) [reply README.md:104]: I did not recall removing this nor edited the ''handling'' section... What could have happened?

Comment (RocketUnicorn) [reply README.md:104]: @nnadeau any idea ?

Comment (mdtusz) [reply README.md:104]: It was Nic's commit here: https://github.com/aon3d/covid-face-shield/pull/12/commits/7848c8cfe22aca8b0f156060050f3739aff7385d 

You didn't do it!

**Review by nugvivi — APPROVED** [review]

**Review by engnadeau — APPROVED** [review]

**Pull request merged**